bluetooth_mesh/gatt.py

```diff
@@ -80,21 +86,51 @@ def read(self, offset=0):
         return self._object.ReadValue(options)
 
     def write(self, value, offset=0):
-        value = GLib.Variant.new_array(
-            None,
-            [GLib.Variant.new_byte(b) for b in value])
-
-        options = {}
-        if offset is not None:
-            options['offset'] = GLib.Variant.new_uint16(offset)
+        if self.write_io:
```

Comment:
I think `write_io` could be a property.

Reply:
it will not be used by other modules. how about just change the the name to `_write_channel`?

Reply:
:+1: about the name. The remark about property was about lazy initialization + more straightforward code in read/write functions. The property can remain protected.

Reply:
yes, get it. thanks.